frontend/src/theme/components/Toast.ts

```diff
@@ -0,0 +1,70 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+export const Toast: ComponentMultiStyleConfig = {
+  parts: ['container', 'icon', 'close', 'content'],
+  baseStyle: {
+    icon: {
+      position: 'absolute',
+      left: '1.125rem',
+      top: '1.125rem',
+      boxSize: '1.25rem',
+    },
+    content: {
+      // NOTE: This is because the outer container already has padding.
+      // So the padding here is icon width + outer padding
+      ml: '1.875rem',
+    },
+    container: {
+      borderRadius: '3px',
+      boxSizing: 'border-box',
+      padding: '1rem',
+      // Padding right is 4 rem (normal padding) + width of the button.
+      // This is to prevent the button overlapping the text on resize.
+      pr: 10,
+      mt: 2,
+      width: {
+        base: 'auto',
+        lg: '42.5rem',
+      },
+      mx: {
+        base: 2,
+        lg: 'inherit',
+      },
+    },
+    close: {
```

Comment:
might need cursor pointer attribute here

Reply:
will check with pearly on this; wrt review: does pearly automatically get notified or should i tag her in the PR also?

Reply:
need to go to chromatic and request from pearly; i've done it for you in this case